content/02.main-text.md

```diff
@@ -144,6 +144,19 @@ We annotated the author list to indicate that author order was partly randomized
 
 ### Discussion
 
+The Deep Review was not the first scholarly manuscript written online via an open collaborative process.
+Interestingly, the collaborative scholarly works (of which we're aware) tend to have dealt with rapidly developing topics.
+In 2013, two dozen mathematicians created the 600-page "HoTT Book" on "Homotopy Type Theory", writing collaboratively in LaTeX on GitHub [@url:https://homotopytypetheory.org/book/; @url:http://math.andrej.com/2013/06/20/the-hott-book/].
```

Comment:
Unless you already checked in a local build, we should confirm after merging that the metadata was extracted correctly for these URL references.